app/views/trends/index.html.erb

```diff
@@ -2,22 +2,9 @@
 <% title_actions display_link_if_authorized(_("Add Trend Counter"), hash_for_new_trend_path),
                  documentation_button('4.1.3Trends') %>
 
-<% if @trends.empty? %>
-  <div class="alert alert-info">
-    <%= alert_close %>
-    <p><strong><%= _("No trend counter defined.") %></strong></p>
-    <p><%= _("To define trend counters, use the Add Trend Counter button.") %><br/>
-      <%= (_("To start collecting trend data, set a cron job to execute 'foreman-rake trends:counter' every Puppet Interval (%s minutes).") % Setting.puppet_interval).html_safe %></p>
```

Comment:
To start collecting trend data, set a cron job to execute 'foreman-rake trends:counter' every Puppet Interval (%s minutes).
To start collecting trend data, set a cron job to execute <span class='black'>foreman-rake trends:counter</span> every Puppet Interval (%s minutes).

@ohadlevy failing to see the diff in messages, the only diff is in the header, but because the solution is running the same command I'm not sure that it would make a difference, i can revert it to be the same as before, let me know.